HintList.py

```diff
@@ -525,20 +525,20 @@ def tokens_required_by_settings(world: World) -> int:
     'ZR Frogs Ocarina Game':                                       (["an #amphibian feast# yields", "the #croaking choir's magnum opus# awards", "the #froggy finale# yields"], "the final reward from the #Frogs of Zora's River# is", ['overworld', 'sometimes']),
     'KF Links House Cow':                                          ("the #bovine bounty of a horseback hustle# gifts", "#Malon's obstacle course# leads to", 'always'),
 
-    'Song from Ocarina of Time':                                   ("the #Ocarina of Time# teaches", None, ['song', 'sometimes']),
-    'Song from Royal Familys Tomb':                                (["#ReDead in the royal tomb# guard", "the #Composer Brothers wrote#"], None, ['song', 'sometimes']),
-    'Sheik in Forest':                                             ("#in a meadow# Sheik teaches", None, ['song', 'sometimes']),
-    'Sheik at Temple':                                             ("Sheik waits at a #monument to time# to teach", None, ['song', 'sometimes']),
-    'Sheik in Crater':                                             ("the #crater's melody# is", None, ['song', 'sometimes']),
-    'Sheik in Ice Cavern':                                         ("the #frozen cavern# echoes with", None, ['song', 'sometimes']),
-    'Sheik in Kakariko':                                           ("a #ravaged village# mourns with", None, ['song', 'sometimes']),
-    'Sheik at Colossus':                                           ("a hero ventures #beyond the wasteland# to learn", None, ['song', 'sometimes']),
+    'Song from Ocarina of Time':                                   ("the #Ocarina of Time# teaches", "the #song# taught by the #Ocarina of Time# is", ['song', 'sometimes']),
```

Comment:
Is the new one more clear than the old one here? They seem to be saying the same thing to me

Reply:
It's more clear by adding song.
Written as is, you can confuse this hint with the Ocarina item hint.

But you can argue that i need to make the Ocarina of Time sometimes hint clearer to - I forgot to check that.
Will go on it when i have more time.